[2025-05-18T01:23:22.595+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api permission to 509
[2025-05-18T01:23:22.596+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api'
[2025-05-18T01:23:22.621+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api permission to 509
[2025-05-18T01:23:22.622+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api'
[2025-05-18T01:23:22.661+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T01:23:21.861002+00:00 [queued]>
[2025-05-18T01:23:22.668+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T01:23:21.861002+00:00 [queued]>
[2025-05-18T01:23:22.668+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2025-05-18T01:23:22.680+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): extract_api> on 2025-05-18 01:23:21.861002+00:00
[2025-05-18T01:23:22.686+0000] {standard_task_runner.py:60} INFO - Started process 209 to run task
[2025-05-18T01:23:22.688+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'extract_api', 'manual__2025-05-18T01:23:21.861002+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmplmitcjlh']
[2025-05-18T01:23:22.691+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask extract_api
[2025-05-18T01:23:22.744+0000] {logging_mixin.py:188} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api permission to 509
[2025-05-18T01:23:22.744+0000] {logging_mixin.py:188} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-18T01:23:21.861002+00:00/task_id=extract_api'
[2025-05-18T01:23:22.746+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_dimensional_model.extract_api manual__2025-05-18T01:23:21.861002+00:00 [running]> on host 5c5bc3724850
[2025-05-18T01:23:23.006+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='extract_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T01:23:21.861002+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-18T01:23:21.861002+00:00'
[2025-05-18T01:23:26.726+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-18T01:23:26.737+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=extract_api, execution_date=20250518T012321, start_date=20250518T012322, end_date=20250518T012326
[2025-05-18T01:23:26.797+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-18T01:23:26.823+0000] {taskinstance.py:3281} INFO - 1 downstream tasks scheduled from follow-on schedule check
